src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -508,21 +494,23 @@ export class BaseEditCourseForm extends React.Component {
                   />
                 </>
               )}
+              isAsync
+              isMulti
               disabled={disabled || !administrator}
-              value={this.state.courseTag}
               optional
-              currentValue={
-                Array.isArray(courseTags)
-                  ? this.courseTagsToSelectValues(courseTags)
-                  : null
-              }
-              setValue={(newValue) => setCourseTags(newValue)}
+              isCreatable
+              styles={{
+                menu: styles => ({ ...styles, zIndex: 5 }),
+                control: styles => ({ ...styles, marginTop: '8px' }),
+              }}
```

Comment:
Can we move the styles in EditCourseForm.scss?

Reply:
+1 on this.

Reply:
also this PR does not change tag field in any visual way